templates/design.md

```diff
@@ -0,0 +1,94 @@
+- Start Date: (fill me in with today's date, YYYY-MM-DD)
+- RFC PR: (leave this empty, to be filled in later)
```

Comment:
To clarify this, is the goal that we would update this field when merging the PR?

Reply:
Correct